### PR TITLE
Fix test2_views unittest

### PR DESCRIPTION
**Pull request by ibmkendrick**
* Tester to add SSL disabled StreamsConnection object into test_config
* modify test2_unicode.py to disable SSL verification

Fixes IBMStreams/streamsx.topology#886

### CODE REVIEW

**Comment by ddebrunner** [comment]
Can one of the admins verify this patch?

**Comment by ddebrunner** [comment]
test now

File: com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/tester.py [hunk 1]
```diff
@@ -269,16 +269,17 @@ def _standalone_test(self, config):
...
         self.sc = config.get(ConfigParams.STREAMS_CONNECTION)
         if self.sc is None:
             # Disable SSL verification
             self.sc = StreamsConnection(username, password)
             self.sc.session.verify = False
+            config[ConfigParams.STREAMS_CONNECTION] = self.sc
+        sjr = stc.submit(stc.ContextTypes.DISTRIBUTED, self.topology, config, username=username, password=password)
+        self.submission_result = sjr
+        if sjr['return_code'] != 0:
+            print("DO AS LOGGER", "Failed to submit job to distributed instance.")
```
Comment (ddebrunner) [inline com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/tester.py]: Please remove.

File: com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/tester.py [hunk 2]
```diff
@@ -269,16 +269,17 @@ def _standalone_test(self, config):
...
-        if sjr['return_code'] != 0:
-            print("DO AS LOGGER", "Failed to submit job to distributed instance.")
-            return False
         self.sc = config.get(ConfigParams.STREAMS_CONNECTION)
         if self.sc is None:
             # Disable SSL verification
             self.sc = StreamsConnection(username, password)
             self.sc.session.verify = False
+            config[ConfigParams.STREAMS_CONNECTION] = self.sc
+        sjr = stc.submit(stc.ContextTypes.DISTRIBUTED, self.topology, config, username=username, password=password)
```
Comment (ddebrunner) [inline com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/tester.py]: Why do I need a StreamsConnection and user/password?

File: com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/tester.py (hunk 1 above)
Comment (ibmkendrick) [inline com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/tester.py]: Remove the print() or remove the return_code check altogether?  I just noticed that the `_streaming_analytics_test` path is not checking the submit return code, so perhaps one should be added there as well.

File: com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/tester.py (hunk 2 above)
Comment (ibmkendrick) [inline com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/tester.py]: The username/password is needed to handle the case when user explicitly specified a username and password on `test()`.  In which case, the `StreamsConnection` would need to be setup with the same credential.

File: com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/tester.py (hunk 1 above)
Comment (ddebrunner) [inline com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/tester.py]: Remove the print, it's left over from debugging right?

Comment (ibmkendrick) [inline com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/tester.py]: ok, will remove.

**Pull request merged**